Mathlib/FieldTheory/Separable.lean

```diff
@@ -496,33 +496,33 @@ the minimal polynomial of every `x : K` is separable.
 We define this for general (commutative) rings and only assume `F` and `K` are fields if this
 is needed for a proof.
 -/
```

Comment:
LGTM, but can you add a sentence in the docstring here that this implicitly assumes that `x` is integral over `F`?